geode-membership/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/GMSMembershipJUnitTest.java

```diff
@@ -328,4 +330,58 @@ public void noDispatchWhenSick() throws MemberShunnedException, MemberStartupExc
     assertThat(spy.getStartupEvents()).isEmpty();
   }
 
+  private void addSurpriseMemberWithVersion(Version version) {
+    MemberIdentifier surpriseMember = myMemberId;
+    surpriseMember.setVmViewId(3);
```

Comment:
It is not clear from looking at this method that it is ok to mutate the object referenced by `myMemberId`. Off-hand it seems like that field refers to a member id representing the "local" member. So either way it seems wrong to modify it here to be the surprise member.

I recommend you construct a new member id here for one-time use.